pkg/operator/sync.go

```diff
@@ -330,17 +327,51 @@ func (optr *Operator) syncRequiredMachineConfigPools(config renderConfig) error
 		}
 		return false
 	}
-	for _, pool := range pools {
-		if err := isMachineConfigPoolConfigurationValid(pool, version.Version.String(), optr.mcLister.Get); err != nil {
-			return fmt.Errorf("pool %s has not progressed to latest configuration: %v, retrying", pool.Name, err)
+
+	var lastErr error
+	if err := wait.Poll(time.Second, 10*time.Minute, func() (bool, error) {
```

Comment:
is 10 too much/too little?

Reply:
looks like it's fine according to Derek so I'd say it's good enough (and what was there before I removed it some months ago)

Reply:
Cross-ref https://github.com/openshift/machine-config-operator/pull/554/files#diff-fa45321336db7ad1cedc28bf643a4f97L326

Reply:
gotcha and thanks @wking !